spec/models/guest_rooms.rb

```diff
@@ -0,0 +1,9 @@
+require "rails_helper"
+
+RSpec.describe GuestRoom, type: :model do
+  it '#create' do
+    @hotel_1 = Hotel.create!(name: "Sheraton", location: "Denver")
+    @room_1 = @hotel_1.rooms.create!(rate: 50, suite: "2 bedroom")
+    @guest_1 = @room_1.guests.create!(name: "Steve", nights: 3)
+
+end
```

Comment:
You don't need to test your `create` method. This is a method that comes from ActiveRecord. Testing that your feature that allows user to create a GuestRoom is the main thing you need to test for this feature.

Reply:
We will want to have relationship tests in here with shoulda_matchers, though... and throughout the other models that you've added relationships to. We should test that `GuestRoom` belongs to guest and belongs to room, that `Room` has many guests through guest_room, and that `Guest` has many rooms through guest_room.